src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -541,6 +541,19 @@
 	 * @return The emoji with the given name (or null if one was not found).
 	 */
 	IEmoji getEmojiByName(String name);
+	
+	/**
+ 	 * This adds a custom emoji to the guild.
+ 	 *
+ 	 * @param name The name, <b>without colons</b> of length 2-32 characters.
+ 	 * @param image The Image of the Emoji.
```

Comment:
Lowercase emoji and image.

Reply:
This was changed to lowercase, github just didn't pick up that it was updated.